ansible/logs.yml

```diff
@@ -24,13 +24,14 @@
         - subjects
       when: "'db' not in exclude_logs_from"
 
-- hosts: shared,core,invokers
+- hosts: controllers,invokers
+  serial: 1
```

Comment:
Wewere able to deploy several controllers on the same machine, because we changed the inventory_name of the machine.
Here it means, that this skript will be executed on all "hosts", which is only one physical host if we have only one machine.
I did not find a solution to execute this on all hosts, but only once if they have the same ansible_host.
So I did the workaround to execute it one after another to avoid race conditions when the machine copys the same log several times.